index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello world we are software developers!</h1>
+    <h1 id="title"> Survey Form </h1>
+    <p id="description"> This is a survey form for microverse trials</p>
+
+    <form id ="survey-form">
```

Comment:
- [ ] It is good practice to use class instead of ID especially in cases like this where the styling can be used on another element. Please change the ID attributes to class